command/quota_apply_test.go

```diff
@@ -97,3 +99,49 @@ func TestQuotaApplyCommand_Good_JSON(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Len(t, quotas, 1)
 }
+
+func TestQuotaApplyNetwork(t *testing.T) {
+	t.Parallel()
+	// srv, _, _ := testServer(t, true, nil)
+	// defer srv.Shutdown()
+
+	type example struct {
+		q *api.QuotaSpec
+		e string
+	}
+
+	intPtr := func(x int) *int {
+		return &x
+	}
+
+	cases := map[string]example{
```

Comment:
It's odd to use a map here to a custom struct here.  The common pattern for table tests is to use structs, something like:

```golang
cases := []struct{
  hcl string
  spec *api.QuotaSpec
  err string
} { ... }
```